File: app/src/main/java/com/malkinfo/shotsvideo/LoginActivity.java
package com.malkinfo.shotsvideo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import android.content.Intent;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);


        Button sendButton = findViewById(R.id.send_button);
        // lambda式
        sendButton.setOnClickListener(v -> {
            Intent intent = new Intent(getApplication(), MainActivity.class);
            startActivity(intent);
    }
}

File: app/src/main/java/com/malkinfo/shotsvideo/LoginActivity.kt
package com.malkinfo.shotsvideo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.malkinfo.shotsvideo.R
import android.content.Intent
import android.view.View
import android.widget.Button
import com.malkinfo.shotsvideo.MainActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        val sendButton = findViewById<Button>(R.id.registration)
        // lambda式
        sendButton.setOnClickListener { v: View? ->
            val intent = Intent(application, MainActivity::class.java)
            startActivity(intent)
        }
    }
}